src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write your code here
+const population = [...document.querySelectorAll('.population')];
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+let total = 0;
+
+const setting = population
+  .map((el) => {
+    return el.textContent;
+  })
+  .map((el) => {
+    return el.replaceAll(',', '');
```

Comment:
The method `replaceAll` is not supported in all browsers. Consider using a more compatible method like `replace` with a global regular expression.